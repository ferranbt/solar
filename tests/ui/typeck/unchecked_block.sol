contract C {
    function a() public {
        unchecked {}

        unchecked {
            unchecked {} //~ ERROR: "unchecked" blocks cannot be nested.
        }
    }
}
