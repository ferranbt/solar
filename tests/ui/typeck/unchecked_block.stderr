error: "unchecked" blocks cannot be nested.
  --> ROOT/tests/ui/typeck/unchecked_block.sol:LL:CC
   |
LL |             unchecked {}
   |             ^^^^^^^^^^^^
   |

error: aborting due to 1 previous error

